transactions/lockedTokens/user/get_total_balance.cdc

```diff
@@ -0,0 +1,81 @@
+import FungibleToken from 0xFUNGIBLETOKENADDRESS
+import FlowToken from 0xFLOWTOKENADDRESS
+import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
+import LockedTokens from 0xLOCKEDTOKENADDRESS
+
+// This script gets the TOTAL number of FLOW an account owns, across unlocked, locked, and staking.
+
+// Adds up these numbers:
+
+// tokens in normal account
+// tokens in normal account staking
+// tokens in normal account delegating
+// tokens in shared account
+// tokens in shared account staking
+// tokens in shared account delegating
+
+
+pub fun main(address: Address): UFix64 {
+
+    var sum = 0.0
+
+    let account = getAccount(address)
+
+    if let vaultRef = account.getCapability(/public/flowTokenBalance)!
+        .borrow<&FlowToken.Vault{FungibleToken.Balance}>() 
+    {
+        sum = sum + vaultRef.balance
+    }
+
+    // Get token balance from the unlocked account's node staking pools
+    let nodeStakerCap = account
+        .getCapability<&{FlowIDTableStaking.NodeStakerPublic}>(
+            FlowIDTableStaking.NodeStakerPublicPath
+        )!
+
+    if let nodeStakerRef = nodeStakerCap.borrow() {
+        let nodeInfo = FlowIDTableStaking.NodeInfo(nodeID: nodeStakerRef.id)
+        sum = sum + nodeInfo.tokensStaked + nodeInfo.totalTokensStaked + nodeInfo.tokensCommitted + nodeInfo.tokensUnstaking + nodeInfo.tokensUnstaked + nodeInfo.tokensRewarded
```

Comment:
```suggestion
        sum = sum 
        	+ nodeInfo.tokensStaked 
        	+ nodeInfo.totalTokensStaked 
        	+ nodeInfo.tokensCommitted 
        	+ nodeInfo.tokensUnstaking 
        	+ nodeInfo.tokensUnstaked 
        	+ nodeInfo.tokensRewarded
```

Maybe also consider putting a function on `NodeInfo` which returns this.

Reply:
yeah that is the plan once we can upgrade